### ISSUE TITLE
Normal clicks register as long clicks

### ISSUE THREAD

**Issue by gilgil28** [mapbox/mapbox-maps-android#1897]
## Environment

- Android OS version: Android 12
- Maps SDK Version: v10.10.0-rc.1

## Observed behavior and steps to reproduce

After a while OnMapLongClickListener is being called for every type of click, including normal click.
OnMapClickListener is no longer called.

In v10.9.1 the issue does not exist

**Comment by kiryldz** [comment]
@gilgil28 can you please provide sample code reproducing this issue? 
No luck with following basic example...
```kotlin
class SimpleMapActivity : AppCompatActivity(), OnMapClickListener, OnMapLongClickListener {

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    val mapView = MapView(this)
    setContentView(mapView)
    mapView.getMapboxMap()
      .apply {
        setCamera(
          CameraOptions.Builder()
            .center(Point.fromLngLat(LONGITUDE, LATITUDE))
            .zoom(9.0)
            .build()
        )
        addOnMapClickListener(this@SimpleMapActivity)
        addOnMapLongClickListener(this@SimpleMapActivity)
      }
  }

  override fun onMapClick(point: Point): Boolean {
    logE("KIRYLDD", "onMapClick")
    return false
  }

  override fun onMapLongClick(point: Point): Boolean {
    logE("KIRYLDD", "onMapLongClick")
    return false
  }

  companion object {
    private const val LATITUDE = 40.0
    private const val LONGITUDE = -74.5
  }
}
```

**Comment by gilgil28** [comment]
@kiryldz 
That code is pretty much what I have. I happens only after doing a lot of long clicking and clicking with more than one finger

**Comment by yunikkk** [comment]
@gilgil28 what devices have you been testing on?

**Comment by gilgil28** [comment]
@yunikkk with Samsung S20 FE 5G

**Comment by kiryldz** [comment]
@gilgil28 still no success, even if clicking a lot with code I shared in https://github.com/mapbox/mapbox-maps-android/issues/1897#issuecomment-1341570370; can you please 
1. try reproduce on another device
2. check out latest `v10.10.0` and not `v10.10.0-rc.1`

**Comment by gilgil28** [comment]
@kiryldz just reproduced it on 10.10.0
It randomly happens (not often though) after a bunch of long clicking and dragging.
I also perform a haptic feedback on long click and I get that immediately as I touch the screen

**Comment by pengdev** [comment]
Hey @gilgil28 since we didn't manage to reproduce it locally, could you help us to narrow down the release that has introduced this issue? i.e. could you try to reproduce it on `v10.10.0-beta.1`?

**Comment by yunikkk** [comment]
@gilgil28 hey! Did you also use view annotations in your app? By any change reproduces when view annotation is removed / hidden?

**Comment by gilgil28** [comment]
@yunikkk I am using point annotations (withIconImage) and line annotations

**Comment by yunikkk** [comment]
@gilgil28 could you check 10.11.0? The issues should be fixed by #1954

**Comment by kiryldz** [comment]
Closing as stale and should be fixed.
